src/playground-code-editor.ts

```diff
@@ -495,9 +510,31 @@ export class PlaygroundCodeEditor extends LitElement {
       }
     });
 
+    if (this.lineWrapping) {
+      cm.on('renderLine', this._onRenderLine);
+    }
+
     this._codemirror = cm;
   }
 
+  private _onRenderLine(
+    editorInstance: Editor,
+    line: CodeMirror.LineHandle,
+    elt: HTMLElement
+  ) {
+    const basePadding = 4;
```

Comment:
Could you add a comment here about what the purpose of this style adjustment logic is? I'm not sure I actually understand what it's doing.

Reply:
@aomarks added a comment, hope it makes a little more sense now 😄

Reply:
Yes, very helpful, thanks!